### ISSUE TITLE
Auto refresh notifications

### ISSUE THREAD

**Issue by ShadowJonathan** [cheeaun/phanpy#544]
Currently using multi-column mode, and one thing i feel missing is auto-refresh on notifications.

Its there for normal columns, but not for notifications, could it be added?

**Comment by cheeaun** [comment]
@ShadowJonathan question, do you have "Auto refresh timeline posts" turned on in Settings?

**Comment by ShadowJonathan** [comment]
Yes, though it doesn't appear to work for notifications, as i have to still click the "new notifications" banner
